resource-directory/service/deviceDirectory.go

```diff
@@ -69,9 +70,11 @@ func decodeContent(content *commands.Content, v interface{}) error {
 }
 
 type Device struct {
-	ID       string
-	Resource *device.Device
-	Metadata *pb.Device_Metadata
+	ID              string
+	Resource        *device.Device
```

Comment:
Bit weird that variable named `Resource` is of type `Device`. :D

Reply:
Hm yes. Jozef? 😂